common/dbc.py

```diff
@@ -173,12 +182,13 @@ def decode(self, x, arr=None, debug=False):
         ival -= (1<<s[2])
 
       # control the offset
-      ival = (ival + s[6])*s[5]
-      if debug:
-        print "%40s  %2d %2d  %7.2f %s" % (s[0], s[1], s[2], ival, s[-1])
+      ival = (ival * s[5]) + s[6]
```

Comment:
It worked only because almost all the signals in the Acura dbc had all their offsets at zero!